network-api/networkapi/campaign/tests.py

```diff
@@ -25,6 +25,7 @@ def test_petition_post_route(self):
             'newsletterSignup': True,
             'comment': 'Test comment',
             'source': 'http://localhost/testing/petition',
+            'locale': 'en'
```

Comment:
We probably want to call this `lang` as well, to stay consistent with what we communicate to.